app/screens/localNode/LocalNodeLoadingPage.js

```diff
@@ -59,6 +61,10 @@ class LocalNodeLoadingPage extends Component<Props, State> {
     }, 10000);
   }
 
+  componentWillUnmount() {
+    this.timer && clearTimeout(this.timer);
```

Comment:
where do you set this.timer? In this version it is always undefined

Reply:
error in merge. fixed.